src/nc_request.c

```diff
@@ -669,6 +669,27 @@ req_recv_done(struct context *ctx, struct conn *conn, struct msg *msg,
         return;
     }
 
+    if (msg->result == MSG_PARSE_ERROR || msg->result == MSG_PARSE_ERROR_WRONG_ARGU_NUM) {
```

Comment:
merge this with noforward?

Reply:
had merged with noforward, also the error_reply func has been merged into reply func,please check the new commit